sphere_lpm_code/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+cmake_minimum_required(VERSION 3.12)
+
```

Comment:
I don't think we need this directory in the main repo.   @michaelchiwere1 please confirm and delete from the branch, if we can.

Reply:
I think we should not need it, indeed.